packages/www/components/Dashboard/Login/index.tsx

```diff
@@ -0,0 +1,171 @@
+import {
+  TextField,
+  Button,
+  Grid,
+  Box,
+  Link as A,
+  Text,
+} from "@livepeer/design-system";
+import { useEffect, useState } from "react";
+import hash from "@livepeer.studio/api/dist/hash";
+import { useRouter } from "next/router";
+import { useHubspotForm } from "hooks";
+import Link from "next/link";
+import BroadcastingProvider from "./BroadcastingProvider";
+
+// The frontend salts are all the same. This could be configurable someday.
+export const FRONTEND_SALT = "69195A9476F08546";
+
+const Login = ({ id, buttonText, onSubmit, loading, errors }) => {
+  const router = useRouter();
+  const [firstName, setFirstName] = useState("");
+  const [lastName, setLastName] = useState("");
+  const [organization, setOrganization] = useState("");
+  const [phone, setPhone] = useState("");
+  const [email, setEmail] = useState("");
+  const [password, setPassword] = useState("");
+
+  const { handleSubmit } = useHubspotForm({
+    portalId: process.env.NEXT_PUBLIC_HUBSPOT_PORTAL_ID,
+    formId: process.env.NEXT_PUBLIC_HUBSPOT_LOGIN_FORM_ID,
+  });
+
+  useEffect(() => {
+    if (router?.query?.email) {
+      setEmail(router.query.email as string);
+    }
+  }, [router?.query?.email]);
+
+  const submit = async (e) => {
+    e.preventDefault();
+    handleSubmit(e);
+
+    const [hashedPassword] = await hash(password, FRONTEND_SALT);
+    // hash password, then
+    onSubmit({
+      email,
+      password: hashedPassword,
+      firstName,
+      lastName,
+      organization,
+      phone,
+    });
+  };
+
+  return (
+    <Box
+      css={{
+        position: "relative",
+        width: "100%",
+      }}>
+      <Text
+        variant="neutral"
+        size={5}
+        css={{
+          maxWidth: 460,
+          textAlign: "center",
+          mx: "auto",
+          mb: "$7",
+        }}>
+        Your home for building onchain video experiences with Livepeer, the
```

Comment:
```suggestion
        Your home for building on-chain video experiences with Livepeer, the
```
Tiny nit

Reply:
There’s actually a concerted effort to drop the hyphen and meme the word word onchain into existence as in “onchain is the next online”. Encourage us all to drop the hyphen!

Reply:
https://twitter.com/search?q=%22Onchain%20is%20the%20next%20online%22&src=typed_query&f=top

Reply:
Sounds good to me!